packages/mui-material/src/Chip/Chip.js

```diff
@@ -63,6 +66,7 @@ const ChipRoot = styled('div', {
         [`& .${chipClasses.deleteIcon}`]:
           styles[`deleteIcon${capitalize(variant)}Color${capitalize(color)}`],
       },
+      { [`& .${chipClasses.rounded}`]: styles.rounded },
```

Comment:
```suggestion
     rounded && styles.rounded
```

(Will need to destructure `rounded` from `ownerState` in line `53` above)